packages/builder/src/pages/builder/app/[application]/_layout.svelte

```diff
@@ -144,17 +148,21 @@
           />
         </span>
         <Tabs {selected} size="M">
-          {#each $layout.children as { path, title }}
-            <TourWrap stepKeys={[`builder-${title}-section`]}>
-              <Tab
-                quiet
-                selected={$isActive(path)}
-                on:click={topItemNavigate(path)}
-                title={capitalise(title)}
-                id={`builder-${title}-tab`}
-              />
-            </TourWrap>
-          {/each}
+          {#key $builderStore?.fonts}
+            <span style="display:contents">
```

Comment:
NAB: Does a <svelte:fragment> work here instead? Or why do we need to insert another DOM node - do we not have other instances of #each blocks inside #key? Or I guess we could move the #key outside the entire Tabs component too.